### PR TITLE
Enable CI using TravisCI

### PR DESCRIPTION
**Pull request by Dexterp37**
This commit will enable linting through flake8 and pytest testing on every PR.

### CODE REVIEW

**Comment by Dexterp37** [comment]
@maurodoglio , any idea why it cannot find `pytest`? The same steps work correctly locally

**Comment by maurodoglio** [comment]
@Dexterp37 It looks like travis is using a cached version of pytest. It could be that an older version is installed on that machine by default.
I would try to specify a minimum version pytest>=3 in the requirements file, see https://github.com/pytest-dev/pytest/issues/1629

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  - "2.7"
+# command to install dependencies
+install:
+  - pip install -r requirements.txt
+  - pip install .
+  - pip install -r test_requirements.txt
+before_script:
+  - flake8 .
+# command to run tests
+script: pytest tests
```
Comment (maurodoglio) [inline .travis.yml]: Missing blank line at the end of file

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+language: python
+python:
+  - "2.7"
+# command to install dependencies
+install:
+  - pip install -r requirements.txt
```
Comment (maurodoglio) [inline .travis.yml]: you can combine the first and the second call to pip:
```bash
pip install -r requirements.txt -r test_requirements.txt
```

**Comment by Dexterp37** [comment]
Thanks @maurodoglio who helped fixing the test errors!

**Pull request merged**